SPECS/util-linux.spec

```diff
@@ -394,6 +394,11 @@ Patch150: 0150-libblkid-minix-Match-minix-superblock-types.patch
 Patch151: 0151-libblkid-minix-Sanity-check-superblock-s_state-for-v.patch
 Patch152: 0152-libblkid-minix-Use-same-checks-for-version-3.patch
 
+# XCP-ng
+Patch1000: 0001-losetup-Add-support-for-logical-block-size.patch
+Patch1001: 0002-losetup-rename-to-sector-size-and-LOG-SEC.patch
+Patch1002: 0003-losetup-properly-use-sector-size-when-create-a-new-d.patch
+Patch1003: 0004-losetup-Fix-typo-for-the-sector-size-docs.patch
```

Comment:
Did you check for other possible fixup patches added to the CentOS 7 package after the addition of these?

Reply:
no, I stuck to the current version